lib/mongo_client.js

```diff
@@ -17,6 +17,83 @@ const connectOp = require('./operations/mongo_client_ops').connectOp;
 const logout = require('./operations/mongo_client_ops').logout;
 const validOptions = require('./operations/mongo_client_ops').validOptions;
 
+const connectSchema = {
+  appname: { type: 'string' },
+  authMechanism: { type: 'string' },
+  authMechanismProperties: { type: 'array' },
+  authSource: { type: 'string' },
+  compression: { type: 'string' }, // alias to compressors
+  connectTimeoutMS: { type: 'number', default: 30000 },
+  heartbeatFrequencyMS: { type: 'number', default: 60000 },
+  j: { type: 'boolean' }, // alias to journal
+  secondaryAcceptableLatencyMS: { type: 'number', default: 15 }, // alias to localThresholdMS
```

Comment:
nit: let's just add these directly in as keys:
```
secondaryAcceptableLatencyMS: { type: 'number', default: 15, alias: 'localThresholdMS' },
```

Even though we don't use it today, it still documents that its an alias and makes it that much easier to use it programmatically in the future.